README.md

```diff
@@ -107,6 +107,18 @@ NOTE
 
 ______________________________________________________________________
 
+### Capturing a replay
+
+For debugging and inspection purposes, it is possible to capture a _replay_, _i.e._ all the minimal information required to replay a series of blocks as they played on the blockchain, which is done with `scripts/capture.pl`.
+
+A typical invocation would be:
```

Comment:
One would run the command scripts/capture.pl from one's own local terminal ?
Also is this the full list of options ?
```bash
--help
--start
--end
--shadownode_fqdn
--ssh_key
--filename
```